src/web/app/pages-instructor/instructor-course-edit-page/instructor-course-edit-page.component.ts

```diff
@@ -403,6 +405,7 @@ export class InstructorCourseEditPageComponent implements OnInit {
       courseId: panelDetail.originalInstructor.courseId,
       requestBody: reqBody,
     }).pipe(finalize(() => panelDetail.editPanel.isSavingInstructorEdit = false)).subscribe((resp: Instructor) => {
+      panelDetail.editPanel.isEditing = false;
```

Comment:
Just a suggestion, you can put this line inside the `finalize` clause so that it is consistent with the loading spinner.